### PR TITLE
Added simple endpoint to view assessment results

### PR DESCRIPTION
**Pull request by oxisto**
- Adds an endpoint to view assessment results
- Refines the query discovery endpoint to include an optional query parameter

### CODE REVIEW

File: cli/commands/assessment/assessment_test.go [hunk 1]
```diff
@@ -0,0 +1,135 @@
+package assessment_test
```
Comment (garuppel) [inline cli/commands/assessment/assessment_test.go:1]: Missing copyright header ;)

File: cli/commands/assessment/assessment_test.go [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
```
Comment (garuppel) [inline cli/commands/assessment/assessment_test.go:2]: Missing copyright header :)

File: cli/commands/assessment/assessment_test.go [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+	}
+
+	// TODO(oxisto): refactor the next lines into a common test login function
+	dir, err = ioutil.TempDir(os.TempDir(), ".clouditor")
+	if err != nil {
+		panic(err)
+	}
+
+	viper.Set("username", "clouditor")
+	viper.Set("password", "clouditor")
```
Comment (garuppel) [inline cli/commands/assessment/assessment_test.go:130]: Currently this is ok, but in the future the password should be read from a config file

Comment (oxisto) [reply cli/commands/assessment/assessment_test.go:130]: Why? these are only for the unit tests.

**Review by garuppel — COMMENTED** [review]
Only minor things

File: voc/voc.go [hunk 4]
```diff
@@ -62,6 +63,17 @@ func (r *Resource) GetType() []string {
 	return r.Type
 }
 
+func (r *Resource) HasType(t string) (b bool) {
```
Comment (garuppel) [inline voc/voc.go:66]: Better use names as variables

**Review by garuppel — COMMENTED** [review]
Use names as variables

**Pull request merged**